test/Libraries/RevitIntegrationTests/ElementTests.cs

```diff
@@ -37,6 +37,20 @@ public void CanDeleteElement()
             // query count node to verify 1 item deleted as a result of the wall deletion. 
             Assert.AreEqual(1, GetPreviewValue("ccd8a5ba37fd4b1297def564392ccf54"));
          }
-        
+
+        [Test]
+        [TestModel(@".\elementJoin.rvt")]
+        public void CanCheckIfElmentsAreJoined()
+        {
+            string samplePath = Path.Combine(workingDirectory, @".\Element\IsJoined_System_Test");
+            string testPath = Path.GetFullPath(samplePath);
+
+            ViewModel.OpenCommand.Execute(testPath);
+
+            RunCurrentModel();
+
+            // query count node to verify 1 item deleted as a result of the wall deletion. 
+            // Assert.AreEqual(1, GetPreviewValue("ccd8a5ba37fd4b1297def564392ccf54"));
```

Comment:
we need to check both elements that are known to be joined and ones that aren't and that the graph runs without warning and results are correct

Reply:
Cant really find a good way of checking if the graph runs without errors from the tests. Will check with the dev guys.

Reply:
ok, since it affects all other open PRs too, will merge this but let's come back to this issue